src/ansible_navigator/command_runner/command_runner.py

```diff
@@ -12,15 +13,16 @@
 PROCESSES = (multiprocessing.cpu_count() - 1) or 1
 
 
-class Command(SimpleNamespace):
+@dataclass
```

Comment:
Can we freeze it?

Reply:
It gets updated after it is instantiated with the outcome of the command:
```
def run_command(command: Command) -> None:
    """run a command"""
    try:
        proc_out = subprocess.run(
            command.command,
            stdout=subprocess.PIPE,
            stderr=subprocess.PIPE,
            check=True,
            universal_newlines=True,
            shell=True,
        )
        command.stdout = proc_out.stdout
    except subprocess.CalledProcessError as exc:
        command.stderr = str(exc.stderr)
```

Reply:
Now noted in the desc with frozen=False.